### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by noahnisbet**
I added a file called KnownDefects.md that has the known problems with our code for the next cohort. This can be updated by anyone if they see a problem that is not included in this document. Also I updated the readme to include the work that the labs31 DS team has done, added the DS team's contacts, and updated the steps for getting the API up locally. If I didn't add something the DS team has worked on please update the readme to include it, and please add anything else that may be necessary. More documentation won't hurt!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,25 +35,46 @@ Labs 30 chose to work asynchronously, yet supporting one another through pair-co
...
+python(3) -m spacy download en_core_web_sm
+uvicorn app.main:app --reload
+
+NOTE: An error may be thrown when running trying to run the app if you have not added a .env file including the Database URL
  ```
  Then open localhost:8000 in your browser. The application should be running. 
 
  ## Contributors
 
+ [Rebecca Wiesenberg, Labs31](https://github.com/rdukewiesenb)
```
Comment (rdukewiesenb) [inline README.md:70]: My name should be changed to "Rebecca Duke Wiesenberg"

**Review by rdukewiesenb — CHANGES_REQUESTED** [review]
The documentation for both files is clear and concise. The only change to make is in the "Contributors" section; my name should be changed to "Rebecca Duke Wiesenberg."

**Review by lucaspetrus — APPROVED** [review]
nice job noah

**Review by lucaspetrus — APPROVED** [review]
update looks good

**Review by codealamode — APPROVED** [review]
Looks great

File: README.md (hunk 1 above)
Comment (codealamode) [reply README.md:70]: Does Noah have to come in and make this change, or is it already made now?  Not sure how a change request on a pull request works / who has to make the change / who has to approve it.

**Review by lucaspetrus — APPROVED** [review]
Thanks noah, my git push is being weird

**Review by codealamode — APPROVED** [review]
looks good

**Review by lucaspetrus — APPROVED** [review]
good to go

**Review by codealamode — APPROVED** [review]

**Review by codealamode — APPROVED** [review]

**Review by babeytapratt — APPROVED** [review]
This looks great.

**Review by lucaspetrus — APPROVED** [review]
trying to approve again

**Review by codealamode — APPROVED** [review]
looks good

**Review by codealamode — APPROVED** [review]

**Pull request merged**